types/options.go

```diff
@@ -53,6 +53,13 @@ func init() {
 		}
 		ReloadConfigurationFileIfNeeded(defaultConfigFile, &defaultStoreOptions)
 	}
+	// reload could set values to empty for run and graph root if config does not contains anything
+	if defaultStoreOptions.RunRoot == "" {
+		defaultStoreOptions.RunRoot = "/run/containers/storage"
```

Comment:
Will this work in rootless mode?

Reply:
At least make these a constant.

Reply:
@rhatdan I think issue only happens for rootful case. But i need to try it myself ill confirm it and make changes.

And sure I agree ill make these constant.